tests/prague/eip7692_eof_v1/eip6206_jumpf/test_jumpf_target.py

```diff
@@ -0,0 +1,106 @@
+"""
+EOF JUMPF tests covering JUMPF target rules.
+"""
+
+import pytest
+
+from ethereum_test_tools import EOFException, EOFTestFiller, StateTestFiller
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .helpers import execute_tests
+from .spec import EOF_FORK_NAME
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-6206.md"
+REFERENCE_SPEC_VERSION = "a1775816657df4093787fb9fe83c2f7cc17ecf47"
+
+pytestmark = pytest.mark.valid_from(EOF_FORK_NAME)
+
+
+@pytest.mark.parametrize(
+    "target_outputs",
+    [NON_RETURNING_SECTION, 0, 2, 4],
+    ids=lambda x: "to-%s" % ("N" if x == NON_RETURNING_SECTION else x),
+)
+@pytest.mark.parametrize(
+    "current_outputs",
```

Comment:
Thinking that changing the word "current" to "source" (so we would have source section and target section) might make it easier to understand. Wdyt @shemnon  ?

Reply:
sounds good.  It also matches the EIP better.